008-sdk-versioning.md

```diff
@@ -0,0 +1,33 @@
+## Revised SDK Versioning Strategy for Iroha Core
+
+### Semantic Versioning
+
+The SDKs will continue to follow semantic versioning with `MAJOR.MINOR.PATCH`. However, the meaning of each part will be related to the current delivery strategy of the Iroha 2 Core. 
+
+- `MAJOR`: Incremented only when there are breaking changes in the SDK itself, regardless of Iroha Core's version.
+- `MINOR`: Incremented for backward-compatible features or enhancements in the SDK.
+- `PATCH`: Incremented for backward-compatible bug fixes or minor updates in the SDK.
+
+### Relationship with Iroha Core
+
+1. Initial Compatibility: Initially, set the SDK versions to align with Iroha Core `20.0.0`, reflecting the current release. For example, the SDKs might start at `2.0.0`.
```

Comment:
I don't see any benefit from it. Moreover, JS SDK packages are already of major version `6.0.0`.

Reply:
That's just an example not even a recommendation to follow

Reply:
Might be less confusing if we use `3.0.0` or `1.0.0` though.

Reply:
I think we still should have some kind of reference to Iroha RC